### ISSUE TITLE
[Package Request] - Kernel backport of 3584718cf2ec bugfix status

### ISSUE THREAD

**Issue by nantiferov** [amazonlinux/amazon-linux-2023#731]
Hi,
Sorry if this is a wrong type of request of wrong repo. I selected what looked like most matching.

**What package is missing from Amazon Linux 2023? Please describe and include package name.**

I would like to know if [this fix](https://git.kernel.org/pub/scm/linux/kernel/git/netdev/net.git/commit/?id=3584718cf2ec) to kernel was backported to 6.1, which is used in AL2023
Bug affects network stack on kernel v6.0 until v6.9 on **arm64/Graviton** hardware and could cause slowdown of network stack, including complete outage refusing new connections.

I wasn't able to find repo from which kernel is build for AL2023, so I created issue here.

**Is this an update to existing package or new package request?**

I guess update of kernel

**Is this package available in Amazon Linux 2? If it is available via external sources such as EPEL, please specify.**

N/A

**Any additional information you'd like to include. (use-cases, etc)**

Basically, in linux kernel version v6.0 was introduced bug in how memory usage is counted (affects only **arm64/Graviton**) and the fix was added in the v6.9 kernel version. Which makes 6.1 kernel used in AL2023 affected too.

Excess memory is not actually being consumed, so it's not a memory leak, though the bug is still fatal since the counter controls Linux's memory pressure logic.

You can check [this Ubuntu bug](https://bugs.launchpad.net/ubuntu/+source/linux/+bug/2045560) timeline for more details.

**Comment by abuehaze14** [comment]
Hi @nantiferov, This fix has been merged in 6.1.90-99.173.amzn2023 kernel, which is in the 2023.4.20230513 release. If you're up to date with AL2023 releases, this should be fixed.

**Comment by nantiferov** [comment]
Thank you very much for details, that's great news.

**Comment by nantiferov** [comment]
Btw, probably you meant "2023.4.202**4**0513" release 
https://docs.aws.amazon.com/linux/al2023/release-notes/relnotes-2023.4.20240513.html

**Comment by abuehaze14** [comment]
nantiferov@ Yes Sorry for that mistake, it's "2023.4.20240513" release.
